src/sql/base/browser/ui/scrollableSplitview/scrollableSplitview.ts

```diff
@@ -314,6 +316,14 @@ export class ScrollableSplitView extends HeightMap implements IDisposable {
 		if (!types.isArray(sizes) && sizes.type === 'distribute') {
 			this.distributeViewSizes();
 		}
+
+		// Re-render the views. Set lastRenderTop and lastRenderHeight to undefined since
+		// this isn't actually scrolling up or down
+		let scrollTop = this.lastRenderTop;
```

Comment:
You shouldn't need this code. this is handled later in the lifecycle of laying out the view